### ISSUE TITLE
vespa-http-client-jar-with-dependencies.jar null pointer exception

### ISSUE THREAD

**Issue by dodysw2** [vespa-engine/vespa#12256]
When reading from stdin, with `--verbose` param, the client throws exception:
```
Exception in thread "main" java.lang.NullPointerException
        at java.base/java.io.File.<init>(File.java:278)
        at com.yahoo.vespa.http.client.runner.Runner.main(Runner.java:89)
```
It seems, there should be null check on `commandLineArgs.getFile()`

https://github.com/vespa-engine/vespa/blob/2686f79b2df3e52e2d43521db4c0b09597c07e73/vespa-http-client/src/main/java/com/yahoo/vespa/http/client/runner/Runner.java#L89

**Comment by jobergum** [comment]
Thank you for the detailed report, we will fix this.

**Comment by olaaun** [comment]
Thanks for pointing this out @dodysw2. A fix has been merged and will be available in a release with version >= 7.181.5
